snap/snapcraft.yaml

```diff
@@ -0,0 +1,117 @@
+name: edgex-app-service-configurable
+base: core18
+version: "v1.1.0-dev.8"
```

Comment:
since you're setting this dynamically below, I think you should use `adopt-info: app-service-config`, and remove the `version` spec entirely since this erronously seems like we're always setting it to `v1.1.0-dev.8` when really this is getting overwritten during the pull stage of the `app-service-config` part.